### PR TITLE
Add multipolygons and geometry collections support to PIA analyssis

### PR DESCRIPTION
**Pull request by AbelVM**
As requested by @andy-esch at https://github.com/CartoDB/crankshaft/issues/164

### CODE REVIEW

File: src/pg/sql/13_PIA.sql [hunk 1]
```diff
@@ -96,27 +96,43 @@ $$ language plpgsql IMMUTABLE;
...
-    SELECT CASE WHEN ST_Within(point,polygon) THEN 1 ELSE -1 END INTO within;
-    SELECT ST_NumInteriorRings(polygon) INTO holes;
-    IF holes > 0 THEN
-        FOR i IN 1..holes
-        LOOP
-            SELECT LEAST(dist, ST_distance(point, ST_InteriorRingN(polygon, i))::numeric) INTO dist;
-        END LOOP;
-    END IF;
+    pols := array_agg((ST_dump(polygon)).geom);
+    FOR j in 1..array_length(pols, 1);
```
Comment (ohasselblad) [inline src/pg/sql/13_PIA.sql]: @AbelVM I think the CI is failing because of the semicolon on this line

**Comment by AbelVM** [comment]
Absolutely. It's Monday in my brain

**Comment by AbelVM** [comment]
Green light! :green_heart:

**Comment by ohasselblad** [comment]
Nice :) Adding to the latest release request -> https://github.com/CartoDB/crankshaft/issues/159

**Pull request merged**